extension/src/popup/helpers/soroban.ts

```diff
@@ -144,7 +144,10 @@ export const getAttrsFromSorobanOp = (
     return null;
   }
 
-  const txAuth = op.auth[0];
+  // disabling op history until we can figure out how to parse the new structure
+  // @ts-ignore
+
+  const txAuth = op.auth;
```

Comment:
similarly, the way these ops are structured is really different now. Disabling until I can dig further into this